### PR TITLE
doc: 6 installation steps for rapidpro cht apk

### PR DESCRIPTION
**Pull request by freefony**

### CODE REVIEW

File: content/en/apps/guides/messaging/gateways/rapidpro_cht_gateway.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#
```
Comment (smbuthia) [inline content/en/apps/guides/messaging/gateways/rapidpro_cht_gateway.md:1]: Add [front matter](https://gohugo.io/content-management/front-matter/) with at least; title, linkTitle, and description.

File: content/en/apps/guides/messaging/gateways/rapidpro_cht_gateway.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+# RapidPro CHT APK with FCM configuration
+
+RapidPro - CHT uses your Android phone to send and receive messages on your behalf.
+
+Due to restrictions Android has on applications that send SMS messages, we cannot distribute the RapidPro - CHT application through the Play Store. You will need to download the application manually and install it using the steps below.
```
Comment (smbuthia) [inline content/en/apps/guides/messaging/gateways/rapidpro_cht_gateway.md:6]: I think it's more accurate to state that the restrictions are from Google on Android applications. I'd rephrase this a bit.
```suggestion
Due to restrictions that Google has placed on Android applications that send SMS messages, RapidPro - CHT applications can no longer be distributed through the Google Play Store. You will need to download the application file and install it using the steps below:
```

File: content/en/apps/guides/messaging/gateways/rapidpro_cht_gateway.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    On your Android device, open your browser and download the app by entering the URL: <code>[https://rapidpro.app.medicmobile.org/android/](https://rapidpro.app.medicmobile.org/android/)</code>
+
+
+
+2. Launch the RapidPro - CHT installer
+
+    Open the notifications shade by dragging from the top and select the downloaded file.
+
+
+![download](rapidpro/apk_download.png "image_tooltip")
```
Comment (smbuthia) [inline content/en/apps/guides/messaging/gateways/rapidpro_cht_gateway.md:21]: The images are not getting rendered. Have a look at the [documentation on image processing](https://gohugo.io/content-management/image-processing/) to guide you on how to add images.

**Review by smbuthia — CHANGES_REQUESTED** [review]
Thanks for adding this guide. Some changes suggested. Are there some additional tips or tricks that you could add on here to flesh it out a little more?

**Comment by garethbowen** [comment]
@freefony Can you please review the suggestions Sam made so we can get these improvements merged?

**Comment by garethbowen** [comment]
@mrjones-plip "randomly" picked you as the new reviewer... please have a look or reassign!

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
Great to see more info on RapidPro  - thanks!

Let's start with three changes and then I can more easily review the PR:
1. rename `rapidpro` directory to `rapidpro_cht_gateway`
2. remove  unused `rapidpro/_index.md` file
3. Fix images so they correctly load in `rapidpro_cht_gateway.md` per [docs on images](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#images). 

This will fix the dupe in the left navigation which shows an mostly empty page when you click. Arrow shows link to actual content:
image

**Comment by garethbowen** [comment]
@freefony Just checking in here - are you free to review the last few changes mrjones suggested?

**Comment by garethbowen** [comment]
@mrjones-plip Can you revive this please and either make the suggested improvements yourself or close the PR as dead?

**Comment by mrjones-plip** [comment]
thanks for the nudge @garethbowen !

@freefony - if you see any mistakes now that this is live, please open a new PR to fix.
